src/widgets/TokenAnnotationWidget.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <table class="selected-tokens">
+    <tr colspan="3" v-if="selectedToken">
+      <a href @click.prevent="onClear">Clear Filter</a>
+    </tr>
+    <tr v-for="token in tokens" :key="token.veRef">
+      <th class="text">{{ token.value }}</th>
+      <td class="parse">{{ token.tag }}</td>
+      <td class="text">{{ token.lemma }}</td>
+    </tr>
+  </table>
+</template>
+
+<script>
+  import gql from 'graphql-tag';
+  import { URN } from '@scaife-viewer/scaife-widgets';
+  import { MODULE_NS, CLEAR_TOKEN } from '@/reader/constants';
+
+  export default {
+    scaifeConfig: {
+      displayName: 'Token Annotations',
+    },
+    methods: {
+      prettyPrint(token) {
+        return JSON.stringify(token, null, 2);
+      },
+      onClear() {
+        this.$store.dispatch(`${MODULE_NS}/${CLEAR_TOKEN}`);
+      },
+    },
+    computed: {
+      selectedToken() {
+        return this.$store.state[MODULE_NS].selectedToken;
+      },
+      // TODO: Dedupe from Reader.vue
+      urn() {
+        return this.$route.query.urn
+          ? new URN(this.$route.query.urn)
+          : this.$store.getters[`${MODULE_NS}/firstPassageUrn`];
+      },
+      // TODO: Dedupe from Reader.vue
+      gqlQuery() {
+        if (this.urn) {
+          return gql`
+          {
+            passageTextParts(reference: "${this.urn.absolute}") {
+              edges {
+                node {
+                  id
+                  ref
+                  tokens {
+                    edges {
+                      node {
+                        veRef
+                        value
+                        lemma
+                        tag
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+          `;
+        }
+        return null;
+      },
+      // TODO: Dedupe from Reader.vue
+      lines() {
+        return this.gqlData
+          ? this.gqlData.passageTextParts.edges.map(line => line.node)
+          : [];
+      },
+      tokens() {
+        const selectedFilter = edge => {
+          return (
+            this.selectedToken === null ||
+            this.selectedToken.veRef === edge.node.veRef
+          );
+        };
+
+        if (this.lines.length > 0) {
+          const tokens = this.lines
+            .map(line => {
+              const { edges } = line.tokens;
+              return edges.filter(selectedFilter).map(edge => {
+                const token = edge.node;
+                return {
+                  veRef: token.veRef,
+                  value: token.value,
+                  lemma: token.lemma,
+                  tag: token.tag,
+                };
+              });
+            })
+            .flat();
+          return [...new Set(tokens)];
+        }
+        return [];
+      },
+    },
+  };
+</script>
+
+<style lang="scss" scoped>
+  @import '../styles/variables';
```

Comment:
We should probably match indentation to the other widgets:

<img height="400" src="https://user-images.githubusercontent.com/629062/81669317-ac085600-940b-11ea-8ae6-7364e28355ed.png" />